com.ibm.team.build.hjplugin/src/main/java/com/ibm/team/build/internal/hjplugin/RTCScm.java

```diff
@@ -1103,8 +1112,13 @@ public boolean checkout(AbstractBuild<?, ?> build, Launcher arg1,
     		throw new AbortException(Messages.RTCScm_checkout_failure4(e.getMessage()));
     	}
 
-
-		return true;
+	}
+	
+	@Override
+	public SCMRevisionState calcRevisionsFromBuild(Run<?, ?> build,
+			FilePath workspace, Launcher launcher, TaskListener listener)
+			throws IOException, InterruptedException {
+		return SCMRevisionState.NONE;
```

Comment:
@jglick you know, I'm not too sure that an implementation of calcRevisionsFromBuild is needed. You see, in RTC you define a build workspace for your Jenkins CI jobs. These are stored on server and RTC in turn then tracks what changes are not accepted in that build workspace from the stream the workspace is following.
This is somewhat analogous to the way cloned git repository can pull changes from the origin, only it happens on the server. The local copy of the workspace in that case is not all that relevant, as the RTC server knows what changes occurred after the build.
See the comment in calcRevisionsFromBuild on https://github.com/zregvart/teamconcert-plugin/blob/master/com.ibm.team.build.hjplugin/src/main/java/com/ibm/team/build/internal/hjplugin/RTCScm.java#L955-L956.